### PR TITLE
Added Migrations and updated Models to resolve name dispute.

### PR DESCRIPTION
**Pull request by IvanMomchilov123**
New Migrations branch.

### CODE REVIEW

File: Travel/Travel/Program.cs [hunk 1]
```diff
@@ -14,6 +18,11 @@ public static void Main(string[] args)
             builder.Services.AddEndpointsApiExplorer();
             builder.Services.AddSwaggerGen();
 
+            builder.Services.AddDbContext<TravelDbContext>(options =>
+            {
+                options.UseSqlServer("Server=DESKTOP-580EIMS;Database=TravelDb;Trusted_Connection=True;TrustServerCertificate=True;");
```
Comment (MartinPetrakiev) [inline Travel/Travel/Program.cs]: Hard coding Connection string is a bad practice. Look for another way

File: Travel/Travel/Models/TripParticipant.cs [hunk 2]
```diff
@@ -3,8 +3,8 @@
     public class TripParticipant
     {
         public int TripId { get; set; }
-        public Trip Trip { get; set; }
+        public Trip RelatedTrip { get; set; }
```
Comment (MartinPetrakiev) [inline Travel/Travel/Models/TripParticipant.cs]: Why are you changing this?

Comment (IvanMomchilov123) [reply Travel/Travel/Models/TripParticipant.cs]: Name dispute. It just wouldn't let me migrate, so I had to rename and the issue stopped.

Comment (MartinPetrakiev) [reply Travel/Travel/Models/TripParticipant.cs]: Check your models and your relationships at 'OnModelCreating' config. This should not happen

File: Travel/Travel/Migrations/TravelDbContextModelSnapshot.cs [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                    b.Property<string>("LastName")
+                        .IsRequired()
+                        .HasColumnType("nvarchar(max)");
+
+                    b.HasKey("UserID");
+
+                    b.ToTable("Users");
+                });
+
+            modelBuilder.Entity("TripUser", b =>
```
Comment (MartinPetrakiev) [inline Travel/Travel/Migrations/TravelDbContextModelSnapshot.cs]: This is additional table for which we don't even have entity for

**Review by MartinPetrakiev — APPROVED** [review]

**Pull request merged**